jsonforms-editor/src/core/renderers/DroppableCategorizationLayout.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * ---------------------------------------------------------------------
+ * Copyright (c) 2020 EclipseSource Munich
+ * Licensed under MIT
+ * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
+ * ---------------------------------------------------------------------
+ */
+
+import {
+  Category,
+  isCategorization,
+  rankWith,
+  StatePropsOfLayout,
+} from '@jsonforms/core';
+import { JsonFormsDispatch, withJsonFormsLayoutProps } from '@jsonforms/react';
+import {
+  AppBar,
+  Card,
+  CardContent,
+  CardHeader,
+  Tab,
+  Tabs,
+  Tooltip,
+} from '@material-ui/core';
+import { PlusOne, Tab as TabIcon } from '@material-ui/icons';
+import React, { useMemo, useState } from 'react';
+
+import { CategorizationLayout } from '../model/uischema';
+import { createCategory } from '../util/generators/uiSchema';
+import { DroppableElementRegistration } from './DroppableElement';
+
+interface DroppableCategorizationLayoutProps extends StatePropsOfLayout {
+  uischema: CategorizationLayout;
+  index?: number;
+}
+
+const DroppableCategorizationLayout: React.FC<DroppableCategorizationLayoutProps> = (
+  props
+) => {
+  const { uischema, schema, path, renderers, cells, index } = props;
+
+  const [currIndex, setIndex] = useState(index);
+
+  const categories = uischema.elements;
+
+  const indicatorColor: 'secondary' | 'primary' | undefined =
+    categories.length === 0 ? 'primary' : 'secondary';
+
+  // fix the current index when categories are deleted
+  if (currIndex && currIndex > categories.length - 1) {
+    setIndex(categories.length === 0 ? undefined : categories.length - 1);
+  }
+
+  // DroppableControl removed itself before dispatching to us, we need
+  // to re-add it for our children
+  const renderersToUse = useMemo(() => {
+    return renderers && [...renderers, DroppableElementRegistration];
+  }, [renderers]);
+
+  const handleChange = (_event: any, value: any) => {
+    if (typeof value === 'number') {
+      setIndex(value);
```

Comment:
We should update the selection from the context here as well, such that the Properties view will show the properties of the Category when a tab is selected. You will also need to stop the event from propagating further. There's an example how to do it in `EditorElement.tsx`.
https://github.com/eclipsesource/jsonforms-editor/blob/master/jsonforms-editor/src/editor/components/EditorElement.tsx#L109

Reply:
Hi Alexandra,

Thanks for your feedback.  All requested changes are applied in the last update. I had to remove also the tooltip for the Add Tab since it was causing a big warning in the JS console and besides we have the description what should be done when there are no tabs created yet.